src/vivarium_public_health/population/mortality.py

```diff
@@ -63,5 +65,11 @@ def mortality_handler(self, event):
             dead_pop['years_of_life_lost'] = self.life_expectancy(dead_pop.index)
             self.population_view.update(dead_pop[['alive', 'exit_time', 'cause_of_death', 'years_of_life_lost']])
 
+    def calculate_mortality_rate(self, index):
+        acmr = self.all_cause_mortality_rate(index)
+        csmr = self.cause_specific_mortality_rate(index, skip_post_processor=True)
+        cause_deleted_mortality_rate = acmr - csmr
+        return pd.DataFrame({'other_causes': cause_deleted_mortality_rate})
```

Comment:
Note, this is *always* a dataframe now, which let me clean up some lines of code elsewhere in the disease stuff that had to handle type conversions.

Reply:
Also, we no longer have a mix of ``death_due_to_other_causes`` and cause names in the ``cause_of_death`` column.